operator/templates/cassandra-yaml.yaml

```diff
@@ -558,6 +558,21 @@ data:
     #    off heap objects
     memtable_allocation_type: {{ .Params.MEMTABLE_ALLOCATION_TYPE }}
 
+    # Limits the maximum Merkle tree depth to avoid consuming too much
+    # memory during repairs.
+    #
+    # The default setting of 18 generates trees of maximum size around
+    # 50 MiB / tree. If you are running out of memory during repairs consider
+    # lowering this to 15 (~6 MiB / tree) or lower, but try not to lower it
+    # too much past that or you will lose too much resolution and stream
+    # too much redundant data during repair. Cannot be set lower than 10.
+    #
+    # For more details see https://issues.apache.org/jira/browse/CASSANDRA-14096.
+    #
+    {{ if .Params.REPAIR_SESSION_MAX_TREE_DEPTH }}
```

Comment:
I think KUDO treats empty string and nil different. So this might be true with current defaults. Setting no default value is different than `""` default value. Same for new params introduced in this PR with default value `""`

Reply:
For future reference: it seems like at least as of right now (KUDO 0.8.0), a template with `if ""` will not trigger the if-condition.

`params.yaml`:
```
  - name: ENABLE_SASI_INDEXES
    description: "Enables SASI index creation on this node. SASI indexes are considered experimental and are not recommended for production use."
    default: ""
```

`cassandra.yaml`:
```
    # Enables SASI index creation on this node.
    # SASI indexes are considered experimental and are not recommended for production use.
    {{ if .Params.ENABLE_SASI_INDEXES }}
    enable_sasi_indexes: {{ .Params.ENABLE_SASI_INDEXES }}
    {{ end }}
```

The line with `enable_sasi_indexes` is not present in the file compiled by KUDO:
```
$ kubectl exec "pod/${kudo_cassandra_instance_name}-${kudo_cassandra_statefulset_name}-${pod}" \
          -n "${kudo_cassandra_instance_namespace}" \
          -c "cassandra" \
          -- \
          bash -c '
            cat "/etc/cassandra/cassandra.yaml" | grep -i sasi
          '
# Enables SASI index creation on this node.
# SASI indexes are considered experimental and are not recommended for production use.
```